packages/venia-ui/lib/components/OrderHistoryPage/index.js

```diff
@@ -1 +1,9 @@
-export { default } from './orderHistoryPage';
+import React from 'react';
+import { Form } from 'informed';
+import OrderHistoryPage from './orderHistoryPage';
+
+export default props => (
```

Comment:
I wish we had a `wrapWithForm` HOC for such cases. A simple HOC that just wraps the given component inside an `informed form`.